modules/renter/hostdb/hostweight.go

```diff
@@ -366,6 +362,22 @@ func (hdb *HostDB) uptimeAdjustments(entry modules.HostDBEntry) float64 {
 	return math.Pow(uptimeRatio, exp)
 }
 
+// decayHostUpOrDowntime decays a host's historic uptime or historic downtime.
+// It also adds the new block height to the historic uptime or historic downtime.
+func decayUptimeOrDowntime(entry *modules.HostDBEntry, scan modules.HostDBScan, recentScan modules.HostDBScan) {
```

Comment:
functions that don't have any receiver should be moved to the top of the file, and then sorted alphabetically among eachother.

Reply:
as pointed out above, this function needs to be moved to scan.go